packages/devextreme/js/__internal/ui/m_validation_engine.ts

```diff
@@ -562,10 +563,11 @@ const ValidationEngine = {
     this.addGroup();
   },
 
-  addGroup(group) {
+  addGroup(group, isRemovable) {
     let config = this.getGroupConfig(group);
     if (!config) {
       config = new GroupConfig(group);
+      config.isRemovable = isRemovable;
```

Comment:
[code style]
Please, name it starting with underscore: `_isRemovable` because it gonna be a private option. 
P.s. groupConfig can be taken by user using [this method](https://js.devexpress.com/jQuery/Documentation/ApiReference/Common/Utils/validationEngine/#getGroupConfiggroup)

Reply:
[9cf2b5d](https://github.com/DevExpress/DevExtreme/pull/27435/commits/9cf2b5dbbeeebb0ab7f8c03aa544c69a576b74ea)